packages/api/src/controllers/playback.ts

```diff
@@ -73,14 +76,25 @@ function newPlaybackInfo(
       url: webRtcUrl,
     });
   }
-  if (recordingUrl) {
+  if (withRecordings) {
     playbackInfo.meta.dvrPlayback = [];
-    playbackInfo.meta.dvrPlayback.push({
-      hrn: "HLS (TS)",
-      type: "html5/application/vnd.apple.mpegurl",
-      url: recordingUrl,
-    });
+    if (!streamRecord) {
+      playbackInfo.meta.dvrPlayback.push({
+        message: "Error: Recording is not enabled for this stream.",
```

Comment:
Instead of `message: "Error: *"`WDYT of just calling the field `error:` with a clean error message? I don't think we should ever use this `message` for anything else.

Reply:
Yeah ok makes sense! I was using `error` before, just felt uncomfortable with having a 2XX response and an error message  inside, but at the end it's just specific to the dvrPlayback, so, changing it to error

Reply:
Yeah, it would be just hiding the fact that it is only used for errors though. In general, backends should not be creating messages to the end-users. This would be just a flag for the frontend or sth (and the 403 on user registration). This will be tech debt if we eventually internationalize our dashboard.